binary_search.py

```diff
@@ -0,0 +1,40 @@
+
+import math
+import unittest
+
+def binary_search(list, target):
+
+    left_element = 0
+    right_element = len(list) - 1
+
+    while left_element <= right_element:
+
+        middle_element = math.floor((left_element + right_element) // 2)
+
+        if list[middle_element] == target:
+            print('Found')
+            return True
+
+        elif list[middle_element] > target:
+            right_element = middle_element - 1
+
+        elif list[middle_element] < target:
```

Comment:
This `elif` could be a else because is the final condition of the initial if and there is no more possibilities.